diff --git a/src/egl/main/eglcurrent.h b/src/egl/main/eglcurrent.h
index e0f6f7b..188aa2d 100644
--- a/src/egl/main/eglcurrent.h
+++ b/src/egl/main/eglcurrent.h
@@ -66,8 +66,7 @@ struct _egl_thread_info {
 static inline EGLBoolean
 _eglIsApiValid(EGLenum api)
 {
-#if HAVE_OPENGL && !DETECT_OS_ANDROID
-   /* OpenGL is not a valid/supported API on Android */
+#if HAVE_OPENGL
    if (api == EGL_OPENGL_API)
       return true;
 #endif
